backtester/report/report_types.go

```diff
@@ -0,0 +1,59 @@
+package report
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/backtester/config"
+	"github.com/thrasher-corp/gocryptotrader/backtester/eventhandlers/statistics"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/kline"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// lightweight charts can ony render 1100 candles
+const maxChartLimit = 1100
```

Comment:
Just a question, will we be generating another chart if we overflow this?

Reply:
Not right now. Is this comment requesting this feature?

Reply:
Nope, just a question. I guess we can add it later anyway on another update if needed. Not in this behemoth.